Gopkg.toml

```diff
@@ -6,6 +6,21 @@
   name = "github.com/aws/aws-sdk-go"
   version = "v1.10.8"
 
+[[constraint]]
```

Comment:
Why?  I would expect master to be the default in a repo with no releases.

Reply:
> Had to fiddle with dep, and upgrade to 0.5.0, as client_golang, weaveworks/common and uber/jaeger-lib were conflicting.

Reply:
What I need here, in the file, is the reason for the constraint, and the conditions under which it can be removed.